common/account_utils/src/validator_definitions.rs

```diff
@@ -152,8 +155,17 @@ impl ValidatorDefinitions {
             .create_new(false)
             .open(&config_path)
             .map_err(Error::UnableToOpenFile)?;
-        Lockfile::new(config_path.join(LOCK_FILE))?;
-        serde_yaml::from_reader(file).map_err(Error::UnableToParseFile)
+        let definitions: Vec<ValidatorDefinition> =
+            match serde_yaml::from_reader(file).map_err(Error::UnableToParseFile) {
+                Ok(t) => t,
+                Err(e) => return Err(e),
+            };
```

Comment:
I think this can be `serde_yaml::from_reader(file).map_err(Error::UnableToParseFile)?`

Reply:
Consult 5a4c3fd.